cfme/tests/v2v/test_post_migrations.py

```diff
@@ -57,6 +58,19 @@ def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini,
     source_view = navigate_to(src_vm_obj, "Details")
     summary = source_view.entities.summary("Properties").get_text_of("Container")
     source_cpu, source_socket, source_core, source_memory = re.findall(r"\d+", summary)
+    snap_name = '{}_snap_'.format(src_vm_obj)
+    snap_desc = 'v2v_snap_test'
+    snapshots_before = src_vm_obj.total_snapshots
+
+    for x in range(3):
+        src_vm_obj.rest_api_entity.snapshots.action.create(name=snap_name + str(x),
+                                                           description=snap_desc, memory=False,)
+    wait_for(
+        lambda: int(src_vm_obj.total_snapshots) == int(snapshots_before) + 3,
```

Comment:
Does this work for both OSP and RHV ?

Reply:
Here the src_vm_obj will always be vpshere vm object and it will work with both OSP and RHV.